scripts/build

```diff
@@ -6,6 +6,9 @@ source $(dirname $0)/images
 
 cd $(dirname $0)/..
 
-cd images
+mkdir -p bin
+[ "$(uname)" != "Darwin" ] && LINKFLAGS="-extldflags -static -s"
+CGO_ENABLED=0 go build -ldflags "-X main.VERSION=$VERSION $LINKFLAGS" -o bin/helm-controller
```

Comment:
... note sure where this came from but now i know why bin/helm-controller kept showing up in my sandbox